b2/account_info/abstract.py

```diff
@@ -137,3 +182,20 @@ def take_large_file_upload_url(self, file_id):
     @abstractmethod
     def clear_large_file_upload_urls(self, file_id):
         pass
+
+    @staticmethod
```

Comment:
- Guido van Rossum, python BDFL, [wrote](https://mail.python.org/pipermail/python-ideas/2012-May/014969.html) this about `staticmethods`: 
> They're basically an accident -- back in the Python 2.2 days when I was inventing new-style classes and descriptors, I meant to implement class methods but at first I didn't understand them and accidentally implemented static methods first. Then it was too late to remove them and only provide class methods.

Generally, whenever there is a temptation to use `@staticmethod`, just use `@classmethod` instead

Reply:
OK.  Good to know.